### DISCUSSION TITLE
How can i make the bot to be used as a waterfall plugin.

### DISCUSSION THREAD

**Discussion by susanthenerd** [PrismarineJS/mineflayer#1569] [General]
I need to use the bot as a waterfall plugin.

**Comment by rom1504** [comment]
What is a waterfall plugin

**Comment by wvffle** [comment]
Simple answer: You can't. Waterfall and mineflayer are two completely different things. 

If you must use a waterfall plugin I advise you to check out the citizens plugin and check if it contains any functionality for moving the NPCs. If not, then check on spigot forums if someone created a plugin for that/wants to do so. You can't simply run nodejs environment directly in java. This wouldn't make much sense to be honest.

If that's not the requirement, you can simply connect to the waterfall server, there shouldn't be any errors with that. If there are - feel free to inform us about them :)
